cnv/cnv-configuring-pxe-booting.adoc

```diff
@@ -0,0 +1,11 @@
+[id="configuring-pxe-booting"]
+= Configuring PXE booting for virtual machines
+include::modules/cnv_document_attributes.adoc[]
+:context: pxe-booting
+toc::[]
+
+include::modules/cnv-networking-glossary.adoc[leveloffset=+1]
+
+include::modules/cnv-pxe-booting-with-mac-address.adoc[leveloffset=+1]
```

Comment:
I followed the actual [link](https://github.com/openshift/openshift-docs/blob/master/modules/cnv_pxebooting.adoc) .
It explains how to create a bridge using ovs-vsctl, which we are not supposed to use anymore (use nmcli or "ip link" instead).

Reply:
> I followed the actual [link](https://github.com/openshift/openshift-docs/blob/master/modules/cnv_pxebooting.adoc) .

Hey, I'm confused about where you're seeing this. cnv_pxebooting.adoc is the old file (present in the CNV 1.4 docs) that is being replaced by cnv-pxe-booting-with-mac-address.adoc in CNV 2.0.

Reply:
Hey Pan
I can't find cnv-pxe-booting-with-mac-address.adoc, neither under modules nor anywhere else in the repo.

Reply:
Hey, it sounds like you are searching within the enterprise-4.1 branch (or the master branch?) but I'm not sure why. All of the files I reference in the assemblies are here in this PR. Does that help?

Reply:
Sorry, my bad. I didn't realize that the referenced files are also part of this PR, so I was looking for them in the branch master branch.